terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -180,26 +192,36 @@ export class TsAutocompleteComponent implements OnInit,
   /**
    * Since the {@link TsFormFieldComponent} is inside this template, we cannot use a provider to pass this component instance to the form
    * field. Instead, we pass it manually through the template with this reference.
+   *
+   * @internal
    */
   public selfReference = this;
 
   /*
    * Implemented as part of {@link TsFormFieldControl}
+   *
+   * @internal
```

Comment:
why is this `@internal` showing as gray instead of red as elsewhere?

Reply:
I have no clue.. testing out docs generation shows that this item is still added to the docs so the flag isn't working as expected in this one spot..

Even changing it to `@ignore` does nothing.. It only seems to work when I create a whole separate JSDoc comment. Which is really odd since these all look identical to me:

<img width="593" alt="Screen Shot 2019-11-26 at 7 52 37 AM" src="https://user-images.githubusercontent.com/270193/69635141-c008f880-1021-11ea-89f1-eb3a11b00d52.png">

I guess I'm going to leave it as is since it appears that it is working most places ¯\_(ツ)_/¯

Reply:
Wow... I had thought it might be a github glitch, but it seems doing something different... interesting..

Reply:
Yeah I'm stumped!